sdk/js/src/Client.ts

```diff
@@ -54,8 +56,24 @@ export class DVCClient implements Client {
 
         const storedAnonymousId = this.store.loadAnonUserId()
         this.user = new DVCPopulatedUser(user, options, undefined, storedAnonymousId ?? undefined)
-
         this.options = options
+        if (!this.options.configCacheTTL) {
+            this.options.configCacheTTL = 604800000
+        }
+
+        const cachedConfig = this.store.loadConfig(this.user)
```

Comment:
the config shouldn't be parsed from the cache if the user ID and fetch date don't match the criteria